clab/exec/exec.go

```diff
@@ -185,10 +174,21 @@ func (ec *ExecCollection) Dump(format string) (string, error) {
 	result := strings.Builder{}
 	switch format {
 	case ExecFormatJSON:
+		// when json format is requested, we check if stdout is a valid json
+		// to nicely display native json output that exec might have produced with `json` tag
+		for _, results := range ec.execEntries {
+			for _, r := range results {
+				if json.Valid([]byte(r.Stdout)) {
+					r.Json = json.RawMessage([]byte(r.Stdout))
+				}
+			}
+		}
```

Comment:
the idea here is that we only populate `execResult.json` when we ask for json formatted output. 
In that case we check if the output has a valid json and populate `r.Json`

Reply:
discard, I have another approach